api/v1/swagger.yaml

```diff
@@ -1790,6 +1790,586 @@ paths:
+      responses:
+        200:
+          description: Success, returns the updated generator.
+          schema:
+            $ref: "#/definitions/GeneratorResponse"
+        401:
+          $ref: "#/responses/401"
+        403:
+          $ref: "#/responses/403"
+        404:
+          $ref: "#/responses/404"
+        default:
+          $ref: "#/responses/genericError"
+
+  # ---------------------------------------------------------------------------
+
+  /generators/{key}/writers:
+    get:
+      security:
+        - jwt: []
+      summary: Get the list of all authorized writers for an generator
+      tags: [ generators, writers ]
+      description: >-
+        Get the list of all authorized writers for an generator. If the Refocus configuration parameter `useAccessToken` is set to `true`, you must include an `Authorization` request header with your [JSON Web Token](https://tools.ietf.org/html/rfc7519) (JWT) as the value. You can get a token using `POST /v1/register` or `POST /v1/tokens`.
+      operationId: getGeneratorWriters
+      parameters:
+        -
+          name: key
+          in: path
+          description: >-
+            The id or name of the generator for which the authorized list of users
+            are to be obtained.
+          required: true
+          type: string
+      responses:
+        200:
+          description: >-
+            Success, returns the users having write permission to the generator.
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/UsersResponse"
+              description: >-
+                The users having write permission to the generator
+        400:
+          $ref: "#/responses/400"
+        404:
+          $ref: "#/responses/404"
+        default:
+          $ref: "#/responses/genericError"
+    post:
+      security:
+        - jwt: []
+      summary: Add one or more users to an generator’s list of authorized writers
+      tags: [ generators, writers ]
+      description: >-
+        Add one or more users to an generator’s list of authorized writers. If the Refocus configuration parameter `useAccessToken` is set to `true`, you must include an `Authorization` request header with your [JSON Web Token](https://tools.ietf.org/html/rfc7519) (JWT) as the value. You can get a token using `POST /v1/register` or `POST /v1/token`.
+      operationId: postGeneratorWriters
+      parameters:
+        -
+          name: key
+          in: path
+          description: >-
+            The id or name of the generator for which the authorized list of users
+            are to be obtained.
+          required: true
+          type: string
+        -
+          name: queryBody
+          description: Request body.
+          in: body
+          required: true
+          schema:
+            type: array
+            items:
+              type: string
+            description: >
+              User name or id
```

Comment:
What happens if User A's ID is the same as User B's Name? Which gets priority? Wouldn't this open the API up to spoofing IDs?